### DISCUSSION TITLE
Getting Started with Reactive Programming

### DISCUSSION THREAD

**Discussion by discordgr** [quarkusio/quarkus#21941] [Q&A]
Hello,

I'm getting started with Reactive programming & Quarkus and i would like some guidance on how to approach the subject. Should i start with RxJava, Vertx or go straight into Mutiny? What would you recommend?

* I'm a junior developer working mainly with Java & Spring boot, so i don't have that much of experience.

**Comment by gastaldi** [comment]
@discordgr welcome! I know it may look complicated to understand it. I'd suggest you these excellent references:

- [Getting Started with Reactive - Quarkus guide](https://quarkus.io/guides/getting-started-reactive) 
- [Reactive Systems in Java book](https://www.oreilly.com/library/view/reactive-systems-in/9781492091714/)

BTW, Mutiny is our preferred API, so we'd recommend using it 😉

**Reply by discordgr** [reply]
@gastaldi Thanks a lot. I'll get on with these.

Concerning mutiny, i was afraid that since it's built on top of other reactive libraries, i would have trouble understanding core concepts.
